client/src/components/SectionTable/CourseInfoButton.js

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+
+class CourseInfoButton extends PureComponent {
+    render() {
+        const { text, icon, redirectLink } = this.props;
+        return (
+            <Button
+                endIcon={icon}
```

Comment:
Let's move the icon to the left side
```suggestion
                startIcon={icon}
```